features/core-download.feature

```diff
@@ -18,6 +18,12 @@ Feature: Download WordPress
     And I run `cd inner && wp core download`
     Then the inner/wp-settings.php file should exist
 
+    When I try `WP_CLI_STRICT_ARGS_MODE=1 wp core download --path=inner`
```

Comment:
I guess I'm still confused on what the bug is. Here's what happens for me locally:

```
local ➜  projects  wp core download --path=test
Creating directory '/Users/danielbachhuber/projects/test/'.
Downloading WordPress 4.8 (en_US)...
Using cached file '/Users/danielbachhuber/.wp-cli/cache/core/wordpress-4.8-en_US.tar.gz'...
Success: WordPress downloaded.
local ➜  projects  wp core download --path=test
Error: WordPress files seem to already be present here.
```

Are you working on Windows? Is this Windows-specific?